WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewModel.swift

```diff
@@ -42,6 +42,21 @@ final class ShippingLabelFormViewModel {
     private(set) var packagesResponse: ShippingLabelPackagesResponse?
     private(set) var selectedPackageID: String?
 
+    /// Customs forms
+    ///
+    private (set) var customsForms: [ShippingLabelCustomsForm] = []
+
+    /// Temporary solution for creating default customs forms.
+    /// When multi-package support is available, we should create separate form for each package ID.
+    ///
+    var defaultCustomsForms: [ShippingLabelCustomsForm] {
+        guard let packageID = selectedPackageID else {
+            return []
+        }
+        let productIDs = order.items.map { $0.productOrVariationID }
+        return [ShippingLabelCustomsForm(packageID: packageID, productIDs: productIDs)]
+    }
```

Comment:
Why not using directly the property `customsForms`?

Reply:
This makes sense - I have updated in 12bc6847ee78f6c1b2e5e8eca6e20392906a732d to reset customs forms to default values when a new package is updated.